operations/finance.md

```diff
@@ -455,52 +489,8 @@ Returns all bills with the specified ids.
             "Notes": "",
             "Number": "29",
             "VariableSymbol": null,
-            "Payments": [
-                {
```

Comment:
Why did we remove this? It would be nice if the response would show sample items or two

Reply:
@siroky I did it in purpose
1) The request contains items: false, so no payments and revenue items
2) The current bills endpoint was/is being kind of miss used. How it works, partners have accounting items and they need to get bill information, but without accounting items. I wanted to hide those items from the response. They will not blindly copy and paste from the documentation and we will not be again in the situation that they pulling items.
3) Accounting items are described on many places in the doc.